repository/repository-web/dist/js/controllers/detailsController.js

```diff
@@ -131,9 +131,7 @@ define(["../init/appController"], function (repositoryControllers) {
                 function (result) {
                   $scope.isLoading = false;
                   $scope.message = result.data.message;
-                  if (result.data.valid) {
-                    $scope.loadDetails();
```

Comment:
What was this originally supposed to do? Reload the details to update the last author/change date?

Reply:
and the properties... e.g. the description.
![reload](https://user-images.githubusercontent.com/57356106/95739717-908bca80-0c8b-11eb-9e91-81a4d002b5ac.jpg)

Reply:
I think it was "something like that". Basically reloading the whole of the model once the "Save" button was pressed, in order to refresh the data. 
So far, I haven't seen any of this requiring reloading the model though, so I figured I'd save us a round-trip to the back-end. 
Other functionalities requiring updating fields, such as releasing the model, etc. do not seem to be impacted.